testing/functional_testing/math_utils/math_utils_test.py

```diff
@@ -22,6 +22,8 @@ def __init__(self, test_dict):
             test_dict["test_exe"],
             test_dict["out_file"],
             test_dict["use_param_file"],
+            test_dict["datm_file"],
+            test_dict["datm_file_url"],
             test_dict["other_args"],
```

Comment:
I'd like to be able to get rid of this by using an extended base class, see above.

Reply:
Done with c65df334.